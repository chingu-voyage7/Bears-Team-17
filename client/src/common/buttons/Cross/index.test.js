import React from 'react';
import { render } from 'react-testing-library';
import CrossButton from '.';

const cross_symbol = String.fromCharCode( 10799);

describe('Cross Button', () => {
  it('renders button', () => {
    const { container, getByText } = render(<CrossButton/>);
    expect(container.firstChild).toMatchSnapshot();
    const cross = getByText(cross_symbol);
    expect(cross).toBeDefined();
  });
});


