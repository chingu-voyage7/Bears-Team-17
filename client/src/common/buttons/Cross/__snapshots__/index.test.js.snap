// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cross Button renders button 1`] = `
<button
  class="sc-bdVaJa cXKJrY"
>
  тип
</button>
`;
